draft-ietf-quic-recovery.md

```diff
@@ -280,23 +296,26 @@ connection's final smoothed RTT value as the resumed connection's initial RTT.
 If no previous RTT is available, or if the network changes, the initial RTT
 SHOULD be set to 100ms.
 
-When a handshake packet is sent, the sender SHOULD set an alarm for the
-handshake timeout period.
+When CRYPTO_HS frames are sent, the sender SHOULD set an alarm for the handshake
+timeout period.
 
```

Comment:
This is going to be messy. The general rule is that some frames require ack, and some don't, and all the frames that require an ack set up the timer. Here you propose that timers would vary based on specific packet content. It smells a lot like "premature optimization".